daemon/algod/api/algod.oas2.json

```diff
@@ -2228,46 +2319,51 @@
     },
     "ErrorResponse": {
       "description": "An error response with optional data field.",
-      "type": "object",
```

Comment:
we could clean this up and make it more uniform, that would require changing every API endpoint to unnest it and use a "$ref" not inside a schema.

Reply:
This is not introducing anything new. This is how it was in the indexer spec, and the sdk implementations were based off this. 
This change here is correcting the discrepancy. 
If you think there is something wrong here, we need to fix it (for both algod and indexer).

Reply:
the change you are proposing will doubly-nest the ErrorResponse type inside two "schema" objects, and I think that will break the spec. because there is already a "schema" wrapped around it in the endpoint definitions above